frontend/src/pages/Home.jsx

```
@@ -88,26 +194,146 @@
+                      <div className="flex items-center">
+                        {[...Array(5)].map((_, index) => (
+                          <svg 
+                            key={index} 
+                            className={`w-5 h-5 ${
+                              index < Math.floor(property.averageRating || 0) 
+                                ? 'text-yellow-400' 
+                                : 'text-gray-300'
+                            }`}
+                            fill="currentColor" 
+                            viewBox="0 0 20 20"
+                          >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)